lib/wireprotocol/shared.js

```diff
@@ -47,9 +47,22 @@ var parseHeader = function(message) {
   };
 };
 
+function applyCommonQueryOptions(queryOptions, cursorState) {
+  if (cursorState.raw) queryOptions.raw = cursorState.raw;
+  if (typeof cursorState.promoteLongs === 'boolean')
```

Comment:
These are even going on to the next line here, so let's do `{}`

Reply:
shall we make a rule that braces are only required if you are breaking the line?

Reply:
I like that rule.

Reply:
OK @daprahamian, I've corrected the cases where if statements were multi-line. Let's not hang up this review on style choices, but rather punt to the team meeting to discuss potentially adding an eslint/prettier rule to catch all of these automatically.